src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -599,7 +578,7 @@ public void should_fail_when_classification_is_supplementary_evidence_with_ocr_d
                     RESPONSE_FIELD_ERRORS,
                     hasItem("The 'attach to case' event is not supported for supplementary evidence with OCR "
                         + "but not containing OCR data")
-        );
+                );
```

Comment:
this formatting change is not required ?

Reply:
Fixed